t/resty/options.t

```diff
@@ -324,3 +326,58 @@ print(str)
 --- out_not_like
 resolver [\s\S]* ipv6=off;
 --- err
+
+
+
+=== TEST 23: specified nameserver overrides system nameservers
+--- opts: --ns 127.0.0.1
+--- src
+local prefix = ngx.config.prefix()
+local conf = prefix.."conf/nginx.conf"
```

Comment:
Please use spaces around binary operators like `..`. Thank you. Please fix other similar places as well.